tests/pipeline-all-amps/repo/pom.xml

```diff
@@ -128,6 +128,15 @@
                                     <overWrite>false</overWrite>
                                     <outputDirectory>${project.build.directory}/amps</outputDirectory>
                                 </artifactItem>
+                                <!-- Do not comment out, as this will cause problems in the Single Pipeline. -->
```

Comment:
Is this necessary for every dependency?!

Reply:
Nope - it's not. It's a copy-paste of previous items. I can remove it and will do it with pleasure 😄